api/src/services/fields.ts

```diff
@@ -834,7 +834,11 @@ export class FieldsService {
 				column = table.increments(field.field);
 			}
 		} else if (field.type === 'string') {
```

Comment:
I'm wondering if the real request here should be to support `char` as a Directus type rather than defaulting to a more generic string 🤔

Reply:
That is an option, although doesn't seem like Knex supports char in that sense.
Alternatively just allow this as an advanced feature to really be able for people to bring their own database without needing to conform to only the generic types allowed by Directus.

Reply:
I'm wondering if we should stop treating `char` as a string, instead return it as `unknown`, and in the case of an `unknown` directus type try defaulting to the database type 🤔 That feels like the most elegant solution here